### PR TITLE
Cancel button fixed (Fixes Issue #1390)

### PR DESCRIPTION
**Pull request by yaoxing1990**
https://rawgit.com/yaoxing1990/yaoxing1990.github.io/xing-fix-cancelbutton/pages/vi/vi-track-progress.html

Checklist:
* Check to see if the following happens: Click "PRs/Issues" -> "Total PRs" (or other buttons), and then hit "cancel" button in prompt window, the window closes and nothing new appears.

### CODE REVIEW

File: pages/vi/vi-track-progress.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
 		console.log(object)
 	
 
-	
+if(user == "" || user == null) 	{	} else {
```
Comment (xyb994) [inline pages/vi/vi-track-progress.html]: I understand that the original script is not formatted well. But I think we could at least be consistent with having an empty space between`if` and `(`. Keeping code style consistent is also essential when we work on a group project.

Leaving the if block empty looks a bit weird.
Would using `if (!(user == "" || user == null))` without `else` be better? Or alternatively use your original if condition with `return;` right after [line 64](https://github.com/yaoxing1990/yaoxing1990.github.io/blob/9ad11852704ddd589848754021012abb561ca4e1/pages/vi/vi-track-progress.html#L64) etc.

File: pages/vi/vi-track-progress.html [hunk 2]
```diff
@@ -91,7 +91,7 @@
...
```
Comment (xyb994) [inline pages/vi/vi-track-progress.html]: Same here regarding code style consistency.

File: pages/vi/vi-track-progress.html [hunk 3]
```diff
@@ -111,7 +111,7 @@
 		console.log(object)
 		//location.href =  object
 
-	
+if(user == "" || user == null) 	{	} else {
```
Comment (xyb994) [inline pages/vi/vi-track-progress.html]: Same here regarding code style consistency.

**Comment by yaoxing1990** [comment]
@xyb994 Modified coding style

**Comment by xyb994** [comment]
@yaoxing1990 I'm looking back at where you made the changes, it makes more sense to have the condition check done right after prompt for the username. It's not necessary to have the rest of function run if the username is not provided.

Overall, this whole file needs clean up (mix of tab, space) and the javascript part could be refactor /simplified.

**Comment by yaoxing1990** [comment]
@xyb994 Fixed, See https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1391/commits/8339c0d6d5034dbb980317098f243764407ec66e

**Pull request merged**